arbitrum-docs/how-arbitrum-works/07-interactive-fraud-proofs.mdx

```diff
@@ -0,0 +1,435 @@
+---
+title: Challenges/:/ Interactive Fraud Proofs
```

Comment:
Any reason to use the symbol `/:/`? I don't recall seeing it anywhere else in the portal.

Reply:
Hmm. Resolve with putting line in quotes, I think the intention was an escape